### DISCUSSION TITLE
Do you have a blog?

### DISCUSSION THREAD

**Discussion by 4-Leafs-Code** [FearlessTech/FearlessTechDiscussions#6] [Announcements]
If you would like to share your blog or other information about yourself, please tell everyone about it here. We might ever showcase you on the FearlessTech site.

**Comment by Goutamxd** [comment]
Hey! That sounds awesome! I'm Goutam , a web designer who loves animations, 3D WebGL, and creating immersive UI/UX experiences. I’ve been diving deep into full-stack development while keeping my passion for design alive.

Currently, I’m participating in a Microsoft Blogathon, writing daily blogs on different topics, and also exploring BrandCraft by Designare to refine my creative skills. Recently, I was part of Google’s GDG Solution Challenge, where my team worked on a healthcare project using MERN Stack & AI. I’ve also explored Three.js, Matter.js, and GLR through hackathons.

I’m always up for tech, design, and creative problem-solving—so if you’re into any of that, let’s connect! 🚀😃
